src/renderers/shared/server/ReactPartialRenderer.js

```diff
@@ -484,6 +484,19 @@ class ReactDOMServerRenderer {
       if (child === null || child === false) {
         return '';
       } else {
+        if (Array.isArray(child)) {
+          var frame = {
+            children: child,
+            childIndex: 0,
+            context: context,
+            footer: '',
+          };
+          if (__DEV__) {
+            frame.debugElementStack = [];
```

Comment:
["setCurrentDebugStack"](https://github.com/facebook/react/blob/master/src/renderers/shared/server/ReactPartialRenderer.js#L66) method will set debugElementStack.length = 1, so need to init the debugElementStack array.